src/runtime/components/forms/Range.vue

```diff
@@ -35,21 +43,30 @@ import colors from '#ui-colors'
 const config = mergeConfig<typeof range>(appConfig.ui.strategy, appConfig.ui.range, range)
 
 export default defineComponent({
-  inheritAttrs: false,
```

Comment:
Why remove this?

Reply:
I added back and used v-bind on both thumbs